src/main/java/net/spy/memcached/protocol/TCPMemcachedNodeImpl.java

```diff
@@ -267,12 +269,28 @@ public final void fillWriteBuffer(boolean shouldOptimize) {
       getWbuf().clear();
       Operation o=getNextWritableOp();
 
-      while(o != null && toWrite < getWbuf().capacity()) {
+      boolean tlsError = false;
```

Comment:
is this not used right now?

Reply:
lol sorry I must have had something weird going on with my GH view when I asked, it is very clearly used below